science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+A comparison with the positivity rate from the [event-related user survey (EDUS)](../2021-08-02-science-blog-3) from the early year 2021 also shows a parallel trend (at a lower level) to the positive share of red warned persons. The difference can be explained as follows: Healthy people and those with milder courses of illness took part in the survey. Those who were very ill probably had other concerns than participating in this survey. For example, on 22/04/2021, about 19% (287/(287+1218)) of CWA data donors had a positive PCR test result at increased risk. For EDUS, it was only about 15% (20/(20+110)). With 6 additional positives, it would also have been 19% here. This suggests that about a quarter of those red warned and positive tested persons had a non-mild course. One could also hypothesise that EDUS participants are more conscientious – after all, they participated in the voluntary survey – and therefore implement more measures to avoid infection.
+
+## 3 Local differences in incidence
+
+When comparing the positivity rates according to risk assessment, it must be examined whether the differences result solely from different incidences.
+
+The following consideration should illustrate this effect:
+
+Suppose we have two regions with very different incidence (and accordingly different prevalence). 
+Both regions have 1,000 inhabitants, but in region A 200 inhabitants are positive at a given time, in region B only 20. In both regions, all inhabitants use the CWA. Due to the high prevalence in region A, 500 CWA users were shown a red tile (and the other 500 a green tile), while in region B only 50 CWA users were shown a red tile (and the other 950 a green tile).
+
+It is further assumed that all CWA users, regardless of their tile colour (risk notification, i.e. risk displayed in the CWA), get tested and that the tile colour is completely insignificant for the risk of infection. (We could assume, for example, that the tile colours are not assigned at all on the basis of [BLE](https://de.wikipedia.org/wiki/Bluetooth_Low_Energy) measured exposures to those who later tested positive, but rather simply randomly assigned depending on the known incidence of a region. (For this, of course, one would have to know in which region the CWA users live.)) Of the 500 CWA users in region A with a red tile, 100 (20%) test positive, and of the 500 with a green tile, 100 also test positive. In region B, 2% of the 50 with a red tile (namely 1 person) and of the 950 with a green tile (namely 19 people) test positive. We said, the same positive rate of 20% and 2% is always assumed for red and green respectively.
```

Comment:
Use English language BLE link https://en.wikipedia.org/wiki/Bluetooth_Low_Energy

Reply:
Thanks for your suggestion, it has been implemented in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5